.github/workflows/release.yml

```diff
@@ -46,22 +46,20 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: install libpq
-        run: sudo apt-get install build-essential libpq-dev
```

Comment:
The base env has built-essential already installed https://github.com/actions/virtual-environments/blob/main/images/linux/ubuntu1804.json#L158